include/RbtAlignTransform.h

```diff
@@ -27,9 +27,22 @@ class RbtAlignTransform: public RbtBaseBiMolTransform {
     static RbtString _COM;
     static RbtString _AXES;
 
+    enum LigandCenterOfMassPlacementStrategy { COM_ALIGN, COM_RANDOM };
+    enum LigandAxesAlignmentStrategy { AXES_ALIGN, AXES_RANDOM };
+
+    struct Config {
+      LigandCenterOfMassPlacementStrategy center_of_mass_placement_strategy;
+      LigandAxesAlignmentStrategy axes_alignment_strategy;
+    };
```

Comment:
I think it is a better practice to provide the default values in the struct definition itself, like this:
```suggestion
    struct Config {
      LigandCenterOfMassPlacementStrategy center_of_mass_placement_strategy {LigandCenterOfMassPlacementStrategy::COM_ALIGN};
      LigandAxesAlignmentStrategy axes_alignment_strategy {LigandAxesAlignmentStrategy::AXES_ALIGN};
    };
```

Reply:
Oh I didn't know that that was possible at all xD Definitely better

Reply:
Any preference on braces or equal syntax? Apparently both can be used. i lean towards equal but that's python talking

Reply:
The modern and recommended way of initializing is with list (or braces, or uniform, all the same) initialization. The '=' is a copy-assignment operator. not very relevant performance-wise in this case, but I'd rather stick to the recommended/preferred current style

Reply:
Applied